R/detect.R

```diff
@@ -6,8 +6,8 @@
 #'
 #'   * A named function, e.g. `mean`.
 #'   * An anonymous function, e.g. `\(x) x + 1` or `function(x) x + 1`.
-#'   * A formula, e.g. `~ .x + 1`. You must use `.x` to refer to the first
-#'     argument. No longer recommended.
+#'   A formula, e.g. `~ .x + 1`. Only recommended if you require backward
+#'     compatibility with older versions of R.
 #'   * A string, integer, or list, e.g. `"idx"`, `1`, or `list("idx", 1)` which
```

Comment:
There's also a mention of "lambda-formula" in the examples that should be removed.